lib/install/react/hello_react.jsx

```diff
@@ -0,0 +1,25 @@
+// Run this example by adding <%= javascript_pack_tag 'hello_react' %> to the head of your layout file,
+// like app/views/layouts/application.html.erb. All it does is render <div>Hello React</div> at the bottom
+// of the page.
+
+import React from 'react'
+import ReactDOM from 'react-dom'
+
+const Hello = props => (
```

Comment:
Functional components have a lot of cachet right now. I think it's a little too fancy for a bare-bones example but we can always fix it later.